quixai/quix-library
# Update local docker environment and Linux guide

- Update local docker setup (Readme and files)
- Update Dockerfile to latest version of quixpythonbaseimage
- Remove kafkadepinstaller.sh as no longer necessary due to updated dockerfile
- Update linux readme
